build/charts/antrea/values.yaml

```diff
@@ -50,6 +50,9 @@ transportInterfaceCIDRs: []
 multicast:
   # -- Names of the interfaces on Nodes that are used to forward multicast traffic.
   multicastInterfaces: []
+  # -- The versions of IGMP queries antrea-agent send to Pods.
+  # Valid versions are 1, 2 and 3.
+  igmpQueryVersions: [1,2,3]
```

Comment:
I suppose the purpose of this PR is to let user to remove some of versions if they don't need it?
I feel it's worthy to add a comment here or on the guide to let user know it can be updated to 1 or 2 or some combinations.